### PR TITLE
🐛Fix `check-types` failure due to new closure version

### PR DESCRIPTION
**Pull request by rsimha**
This PR fixes a failure that snuck in with #28243.

It turns out we were missing a few runtime checks for package upgrades. Fix sent out in #28317.

### CODE REVIEW

File: 3p/twitter.js [hunk 1]
```diff
@@ -16,6 +16,7 @@
 
 // TODO(malteubl) Move somewhere else since this is not an ad.
 
+import {dev} from '../src/log';
```
Comment (jridgewell) [inline 3p/twitter.js:19]: I don't think you can import `log` here, because this is running in a 3p window context. I would say just typecast.

Comment (rsimha) [reply 3p/twitter.js:19]: Done.

**Comment by rsimha** [comment]
FYI @ampproject/build-cop, this should fix the redness on `master`.

**Comment by rsimha** [comment]
[`check-types`](https://travis-ci.org/github/ampproject/amphtml/jobs/685866687#L440) is green, as is [`dist`](https://travis-ci.org/github/ampproject/amphtml/jobs/685866689#L574). Force-merging to unbreak `master` and unblock development.

**Pull request merged**